tools/devctr/poetry.lock

```diff
@@ -2,87 +2,87 @@
 
 [[package]]
 name = "aiohttp"
-version = "3.9.4"
+version = "3.9.5"
```

Comment:
It seems something is off with the update? The devctr sanity check seems to fail

Reply:
Yeah, I kinda don't understand how this lockfile works tbh... if external factors can break it, then what is the point of having a lockfile? lol

Reply:
fixed it by regenerating the lockfile